components/atom/progressBar/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/react-atom-progress-bar",
-  "version": "1.10.0",
```

Comment:
this should be set by the automatic release process, did you create a commit adding something to the breaking changes question?

Reply:
Yes, I did, but @andresin87 told me it wasn't automatic so now I do not know what to do xD

Reply:
It should be working, tell me if not

Reply:
The automatic release process upgrades the X.[number].0 version. As long as I know, this it the requirement to create a new major

Reply:
it changes the X if you fill the breaking changes question while running sui-studio commit command